Cargo.toml

```diff
@@ -18,7 +18,8 @@ path = "src/lib.rs"
 
 [dependencies]
 log = "^0.4"
-bitcoin = { version = "^0.29", features = ["serde"] }
+bitcoin = { version = "^0.30", features = ["serde"] }
+bitcoin-private = "0.1.0"
```

Comment:
to access method to convert bytes to hex as suggested here https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/286#issuecomment-1483378019